app/models/approval_request.rb

```diff
@@ -3,4 +3,12 @@ class ApprovalRequest < ApplicationRecord
   enum :state => [:undecided, :approved, :denied]
 
   belongs_to :order_item
+
```

Comment:
@lindgrenj6 Should these be in the service object that handles the transition and not in the models, since the models are supposed to be practically empty?